pkg/api/controller.go

```diff
@@ -3006,6 +3011,57 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email string, token string) error {
+	return c.Emailer.SendEmail([]string{email}, token, token, nil)
+}
+
+func (c *Controller) PasswordForgot(w http.ResponseWriter, r *http.Request, body PasswordForgotJSONRequestBody) {
```

Comment:
I know it is a bit late but 'passwordForgot' (operation id) should be 'forgotPassword' - it is API and I think we better have 'ForgotPassword' as a method and not the other way around.
It is hard to change API later and I think we will be sorry to lookup PasswordForgot later.